tests/integ/modin/test_apply_snowflake_cortex_functions.py

```diff
@@ -82,23 +89,145 @@ def test_apply_snowflake_cortex_sentiment_df(session):
     running_on_jenkins(),
     reason="TODO: SNOW-1859087 snowflake.cortex.sentiment SSL error",
 )
-@sql_count_checker(query_count=0)
 @pytest.mark.parametrize(
-    "is_series, operation",
+    "is_series, operation, query_count",
+    [
+        param(
+            True,
+            (lambda s: s.apply(ClassifyText, categories=["travel", "cooking"])),
+            1,
+            id="series_cortex_classify_text_kwargs",
+        ),
+        param(
+            False,
+            (lambda df: df.apply(ClassifyText, categories=["travel", "cooking"])),
+            2,
+            id="df_cortex_classify_text_kwargs",
+        ),
+    ],
+)
+def test_apply_snowflake_cortex_classify_text(
+    session, is_series, operation, query_count
+):
+
+    # TODO: SNOW-1758914 snowflake.cortex.sentiment error on GCP
+    with SqlCounter(query_count=0):
+        if session.connection.host == "sfctest0.us-central1.gcp.snowflakecomputing.com":
+            return
+
+    with SqlCounter(query_count=query_count):
+        content = "One day I will see the world."
+
+        modin_input = (pd.Series if is_series else pd.DataFrame)([content])
+        text_class = operation(modin_input)
+        if is_series:
+            text_class_label = text_class.iloc[0]["label"]
+        else:
+            text_class_label = text_class[0][0]["label"]
+        assert text_class_label == "travel"
+
+
+@pytest.mark.skipif(
+    running_on_jenkins(),
+    reason="TODO: SNOW-1859087 snowflake.cortex.sentiment SSL error",
+)
+@pytest.mark.parametrize(
+    "is_series, operation, query_count",
+    [
+        param(
+            True,
+            (lambda s: s.apply(Translate, from_language="en", to_language="de")),
+            1,
+            id="series_cortex_translate",
+        ),
+        param(
+            False,
+            (lambda df: df.apply(Translate, from_language="en", to_language="de")),
+            2,
+            id="df_cortex_translate",
+        ),
+    ],
+)
+def test_apply_snowflake_cortex_translate(session, is_series, operation, query_count):
+
+    # TODO: SNOW-1758914 snowflake.cortex.sentiment error on GCP
+    with SqlCounter(query_count=0):
+        if session.connection.host == "sfctest0.us-central1.gcp.snowflakecomputing.com":
+            return
+    with SqlCounter(query_count=query_count):
+        content = "Good Morning"
+
+        modin_input = (pd.Series if is_series else pd.DataFrame)([content])
+        res = operation(modin_input)
+        if is_series:
+            translated_text = res.iloc[0]
+        else:
+            translated_text = res[0][0]
+        assert translated_text == "Guten Morgen"
```

Comment:
```suggestion
        assert translated_text.lower() == "guten morgen"
```

Let's be case-insensitive to reduce the chance that the model does something unexpected.

Reply:
Good point, it's now updated!